BGL/include/CGAL/boost/graph/named_params_helper.h

```diff
@@ -356,6 +357,22 @@ struct GetPolygonSoupGeomTraits
                    > ::type                                                         type;
 };
 
+namespace internal {
+template <class Point, int d = CGAL::Ambient_dimension<Point>::value>
+struct Vector {
```

Comment:
This name, although in `internal`  looks too generic. @MaelRL ??

Reply:
Vector_for_point or Vector_matching_point?

Reply:
Why do you need it at all?

Reply:
I use the Point_set_processing_3_np_helper to get the PointMap. The helper only worked for Point_3 and I made it general to work with any dimension. As it also handles and defines normal maps, a vector type is required.

Alternatively, I can also disable the NormalMap definition in the helper, in case the point dimension is not 3.

Reply:
The method itself does not need the Vector type.

Reply:
Vector_matching_point